docs/src/main/asciidoc/spring-cloud-stream.adoc

```diff
@@ -1694,22 +1694,23 @@ The user must check for the binder type and then apply the necessary customizati
 
 Here is an example of providing a `BinderCustomizer` bean.
 
-```
+[source, java]
+----
 @Bean
 public BinderCustomizer binderCustomizer() {
     return (binder, binderName) -> {
         if (binder instanceof KafkaMessageChannelBinder) {
-            ((KafkaMessageChannelBinder) binder).setRebalanceListener(...);
+            ((KafkaMessageChannelBinder) binder).setRebalanceListener(/*...*/);
         }
         else if (binder instanceof KStreamBinder) {
-            ...
+            // ...
         }
         else if (binder instanceof RabbitMessageChannelBinder) {
-            ...
+            // ...
```

Comment:
It is implied that this is a pseudo code (incomplete code). There is no reason to escape it.